components/root/root.jsx

```diff
@@ -323,17 +323,29 @@ export default class Root extends React.PureComponent {
         GlobalActions.redirectUserToDefaultTeam();
     }
 
+    initiateMeRequests = async () => {
+        await Promise.all([
+            this.props.actions.getClientConfig(),
+            this.props.actions.getLicenseConfig(),
+        ]);
+
+        let isMeLoaded = false;
+        if (document.cookie.includes('MMUSERID=')) {
+            const dataFromLoadMe = await this.props.actions.loadMe({isLoggedIn: false});
```

Comment:
Should `.loadMe({isLoggedIn: false})` be `loadMe(false)` instead. AFAICT, the action that's being bound is the one with signature `export function loadMe(isLogginIn = false): ActionFunc {` from `packages/mattermost-redux/src/actions/users.ts`

Reply:
oh yeah, sorry passed the correct argument now.

Reply:
✅  Updated as per request/discussion